packages/imperative/src/config/src/Config.ts

```diff
@@ -179,7 +179,9 @@ export class Config {
                 exists: false,
                 properties: Config.empty(),
                 global: layer === Layers.GlobalUser || layer === Layers.GlobalConfig,
-                user: layer === Layers.ProjectUser || layer === Layers.GlobalUser
+                user: layer === Layers.ProjectUser || layer === Layers.GlobalUser,
+                ignoreErrors: process.argv.includes("--help") || process.argv.includes("--version")
```

Comment:
This works for full option names.
should we have this working with aliases as well?
`-V` and `-h` 😋

Reply:
We could reference constants defined in `packages/imperative/src/constants/src/Constants.ts` - `HELP_OPTION`, `HELP_OPTION_ALIAS`, etc.